DOCUMENTATION.md

```diff
@@ -1,16 +1,19 @@
 ## Version: 2.11.0
 
-The SAP Cloud SDK for JavaScript and TypeScript consists of multiple libraries to enable you to write side-by-side extensions on SAP Business Technology Platform.
+The SAP Cloud SDK for JavaScript (and TypeScript) consists of multiple libraries to enable you to write side-by-side extensions on SAP Business Technology Platform.
 
-You can find tutorials that showcase some of the typical use cases on [developers.sap.com](https://developers.sap.com/tutorial-navigator.html?tag=products:technology-platform/sap-cloud-sdk/sap-cloud-sdk&tag=topic:javascript).
-If you have questions, you can reach out to us on [StackOverflow](https://stackoverflow.com/questions/tagged/sap-cloud-sdk?tab=Newest) using the tag `sap-cloud-sdk` and on [answers.sap.com](https://answers.sap.com/tags/73555000100800000895) using the tag `SAP Cloud SDK`.
-Also, you can browse our [release notes](https://help.sap.com/doc/2324e9c3b28748a4ae2ad08166d77675/1.0/en-US/js-index.html) to check out what's new in each version.
-
-### More Helpful Links
-
-- [All versions of this documentation](https://help.sap.com/viewer/product/SAP_CLOUD_SDK/1.0/en-US)
-- [Product page of the SAP Cloud SDK](https://developers.sap.com/topics/cloud-sdk.html)
```

Comment:
please check the markdown syntax, the links appear broken (at least on github)

<img width="1172" alt="Screenshot 2023-01-17 at 14 42 38" src="https://user-images.githubusercontent.com/2292245/212914408-4dfd2cf9-6c38-46c5-b750-1172b5e8482c.png">

Reply:
looks like the br tags are an issue for github, not sure why